hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -820,6 +820,7 @@ message SnapshotDiffJobProto {
   optional bool forceFullDiff = 8;
   optional uint64 totalDiffEntries = 9;
   optional string message = 10;
+  optional bool forceNonNativeDiff = 11;
```

Comment:
Up to you if you want to do the rename or not.

```suggestion
  optional bool diffDisableNativeLibs = 11;
```

Reply:
done